waspc/data/Generator/templates/server/src/routes/auth/signup.js

```diff
@@ -0,0 +1,10 @@
+import { createNewUser } from '../../core/auth.js'
+import { handleRejection } from '../../utils.js'
+
+export default handleRejection(async (req, res) => {
+  const userFields = req.body || {}
+
+  await createNewUser(userFields)
```

Comment:
Could it be problematic that any field is forwarded here? Should we limit it to email / password? Maybe it is ok for now, but we could leave a TODO hm.